homeassistant/components/reolink/strings.json

```
@@ -335,6 +335,83 @@
           "off": "Awake",
           "on": "Sleeping"
         }
+      },
+      "crossline_person": {
+        "name": "Crossline {zone_name} person",
+        "state": {
+          "off": "[%key:component::binary_sensor::entity_component::gas::state::off%]",
```

```suggestion
          "off": "[%key:component::binary_sensor::entity_component::occupancy::state::off%]",
```
can we use `occupancy`? I know `occupancy` uses `gas`, but if we ever update either `gas` or `occupancy`, then this points at the correct one.

---

Done, I have changed it consitantly throughout reolink

---

Reverting since this is apperently not allowed, see CLI error:
` [TRANSLATIONS] Lokalise supports only one level of references: "strings.json::entity::binary_sensor::face::state::off" should point to directly to "component::binary_sensor::entity_component::gas::state::off"`